session/bootstrap.go

```diff
@@ -2129,8 +2110,10 @@ func doDMLWorks(s Session) {
 }
 
 func mustExecute(s Session, sql string, args ...interface{}) {
-	ctx := kv.WithInternalSourceType(context.Background(), kv.InternalTxnBootstrap)
+	ctx, cancel := context.WithTimeout(context.Background(), 75*time.Second)
```

Comment:
How is 75 seconds decided?

Reply:
the ttl of the ddl owner key is 60s, we should wait it is expired and have some time to finish it

Reply:
It shouldn't be a magic number anyway. Please add a variable and comments for it.

Reply:
ok